examples/basic/package.json

```diff
@@ -13,5 +13,6 @@
   "devDependencies": {
     "prettier": "^2.5.1",
     "turbo": "latest"
-  }
+  },
+  "packageManager": "npm@8.5.5"
```

Comment:
If the examples all still run properly, I'm fine leaving this in here, as it definitely makes debugging easier.

Reply:
I've actually already removed it, this was from a WIP commit that accidentally grabbed too much.